packages/appkit/src/client.ts

```diff
@@ -2022,16 +2015,39 @@ export class AppKit {
     )
   }
 
+  private getUnsupportedNetwork(caipNetworkId: CaipNetworkId) {
+    return {
+      id: caipNetworkId.split(':')[1],
+      caipNetworkId,
+      name: ConstantsUtil.UNSUPPORTED_NETWORK_NAME,
+      chainNamespace: caipNetworkId.split(':')[0],
+      nativeCurrency: {
+        name: '',
+        decimals: 0,
+        symbol: ''
+      },
+      rpcUrls: {
+        default: {
+          http: []
+        }
+      }
+    } as CaipNetwork
+  }
+
```

Comment:
this does not require internal AppKit state, can we move to utils?

Reply:
We need to set some values dynamically like `id`, `caipNetworkId`, `name` and `chainNamespace`, so I'd prefer to keep rest here as well